src/test/resources/application-test.yml

```diff
@@ -0,0 +1,20 @@
+# Config overrides common to all tests should go here.
+# All tests should enable the "test" profile in order to load these property overrides.
```

Comment:
[optonal]: not sure the best practice for naming the file and profile name. Do you think we should name as 'test' or 'testBase', or something else?

Reply:
Yeah, I'm not in love with `test` but I think for profile names, brevity is slightly more important than specificity. So I'll stick with `test` for now, we can always adapt later if it's not clear.